udata/templates/territories/search-result.html

```diff
@@ -1,14 +1,21 @@
 {% cache cache_duration, 'territory-search-result', territory.id, g.lang_code %}
 {% set logo=territory.logo_url() or theme_static('img/placeholder_territory.png') %}
+{% if territory.level_name == 'town' %}
```

Comment:
I think you can use GeoLevels and use the same pattenr as: https://github.com/opendatateam/udata/blob/master/udata/core/spatial/models.py#L130-L138 :
- loaded once a startup
- cached by language